e2e cluster didn't start because of certificates

Cluster failed to start with the following error:

    00:50:00 e2e-scalability-master-https e2e-scalability 0.0.0.0/0  tcp:443          e2e-scalability-master
    00:52:07 /jenkins-master-data/jobs/kubernetes-e2e-gce-scalability/workspace/kubernetes/hack/e2e-internal/../../cluster/../cluster/../cluster/gce/util.sh: line 523: cd: easy-rsa-master/easyrsa3: No such file or directory
    00:52:07 === Failed to generate certificates: Aborting ===
    00:52:07 2015/12/08 00:52:07 e2e.go:309: Error running up: exit status 2
    00:52:07 2015/12/08 00:52:07 e2e.go:305: Step 'up' finished in 2m36.843513874s
    00:52:07 2015/12/08 00:52:07 e2e.go:141: Error starting e2e cluster. Aborting.

http://kubekins.dls.corp.google.com/view/Critical%20Builds/job/kubernetes-e2e-gce-scalability/3336/console

@kubernetes/goog-testing

--- a/cluster/gce/util.sh
+++ b/cluster/gce/util.sh
@@ -518,7 +518,7 @@ function create-certs {
 
   # Note: This was heavily cribbed from make-ca-cert.sh
   (cd "${KUBE_TEMP}"
-    curl -L -O https://storage.googleapis.com/kubernetes-release/easy-rsa/easy-rsa.tar.gz > /dev/null 2>&1
+    curl -L -O --connect-timeout 20 --retry 6 --retry-delay 2 https://storage.googleapis.com/kubernetes-release/easy-rsa/easy-rsa.tar.gz > /dev/null 2>&1
     tar xzf easy-rsa.tar.gz > /dev/null 2>&1
     cd easy-rsa-master/easyrsa3
     ./easyrsa init-pki > /dev/null 2>&1

